source/javascripts/directives/image_loader.coffee

```diff
@@ -0,0 +1,15 @@
+EGA.directive 'imageSuggestions', ->
```

Comment:
You could make it a directive that you can provide "alternative" content, e.g: `<span class="category">{{ award.category }}</span>` and it simply chooses what to display, the image tag or the alternative content. For this you need to use the `transclude` property of directives in combination with [ngTransclude](https://docs.angularjs.org/api/ng/directive/ngTransclude) which is pretty powerful.

```javascript
restrict: 'E'
transclude: true
```

you can read that up here: https://docs.angularjs.org/guide/directive
search for "transclude" on that page …